### PR TITLE
feat: GCP multi project Terraform

### PR DESCRIPTION
**Pull request by jon-stewart**
## Summary

https://developer.hashicorp.com/terraform/language/v1.1.x/meta-arguments/for_each

## How did you test this change?

unit and integration test

`lacework generate cloud-account gcp --projects project1 --projects project2 --projects project3`

## Issue

https://lacework.atlassian.net/browse/GROW-1502

### CODE REVIEW

**Review by afiune — COMMENTED** [review]
I tried to fix this but the generated code is not valid, I couldn't check why.

File: cli/cmd/generate_gcp.go [hunk 1]
```diff
@@ -441,6 +442,11 @@ func initGenerateGcpTfCommandFlags() {
 		"use_pub_sub",
 		false,
 		"use pub/sub for the audit log data rather than bucket")
+	generateGcpTfCommand.PersistentFlags().StringArrayVar(
```
Comment (afiune) [inline cli/cmd/generate_gcp.go:445]:
```suggestion
	generateGcpTfCommand.PersistentFlags().StringSliceVar(
```

Comment (afiune) [reply cli/cmd/generate_gcp.go:445]: https://pkg.go.dev/github.com/spf13/pflag#StringSliceVar

**Review by afiune — COMMENTED** [review]
This is covering non-interactive mode. Can we also cover interactive mode? that is, survey questions?

File: integration/test_resources/help/generate_cloud-account_gcp [hunk 2]
```diff
@@ -45,6 +45,7 @@ Flags:
       --output string                                 location to write generated content (default is ~/lacework/gcp)
       --prefix string                                 prefix that will be used at the beginning of every generated resource
       --project_id string                             specify the project id to be used to provision lacework resources (required)
+      --projects stringArray                          list of project IDs to integrate with (project-level integrations)
```
Comment (afiune) [inline integration/test_resources/help/generate_cloud-account_gcp:48]:
```suggestion
      --projects strings                              list of project IDs to integrate with (project-level integrations)
```

**Comment by jon-stewart** [comment]
```
> terraform validate
╷
│ Error: Invalid value for input variable
│
│   on main.tf line 29, in module "gcp_project_audit_log":
│   29:   service_account_name         = module.gcp_project_level_config.service_account_name
│
│ The given value is not suitable for module.gcp_project_audit_log.var.service_account_name declared at .terraform/modules/gcp_project_audit_log/variables.tf:39,1-32: string required.
╵
╷
│ Error: Invalid value for input variable
│
│   on main.tf line 30, in module "gcp_project_audit_log":
│   30:   service_account_private_key  = module.gcp_project_level_config.service_account_private_key
│
│ The given value is not suitable for module.gcp_project_audit_log.var.service_account_private_key declared at .terraform/modules/gcp_project_audit_log/variables.tf:45,1-39: string required.
╵
```

**Comment by jon-stewart** [comment]
Requires:
```
    service_account_name         = module.gcp_project_level_config[each.key].service_account_name
    service_account_private_key  = module.gcp_project_level_config[each.key].service_account_private_key
```

**Review by dmurray-lacework — APPROVED** [review]

**Pull request merged**